src/EnergyPlus/HeatBalFiniteDiffManager.hh

```diff
@@ -294,19 +305,18 @@ namespace HeatBalFiniteDiffManager {
 
 struct HeatBalFiniteDiffMgr : BaseGlobalStruct
 {
-    Array1D_string const cCondFDSchemeType = Array1D_string(2, {"CrankNicholsonSecondOrder", "FullyImplicitFirstOrder"});
-
     Array1D<Real64> SigmaR; // Total Resistance of construction layers
     Array1D<Real64> SigmaC; // Total Capacitance of construction layers
 
     Array1D<Real64> QHeatInFlux;  // HeatFlux on Surface for reporting
     Array1D<Real64> QHeatOutFlux; // HeatFlux on Surface for reporting
 
-    int CondFDSchemeType = HeatBalFiniteDiffManager::FullyImplicitFirstOrder; // solution scheme for CondFD - default
-    Real64 SpaceDescritConstant = 3.0;                                        // spatial descritization constant,
-    Real64 MinTempLimit = -100.0;                                             // lower limit check, degree C
-    Real64 MaxTempLimit = 100.0;                                              // upper limit check, degree C
-    int MaxGSiter = 30;                                                       // maximum number of Gauss Seidel iterations
+    HeatBalFiniteDiffManager::CondFDScheme CondFDSchemeType =
+        HeatBalFiniteDiffManager::CondFDScheme::FullyImplicitFirstOrder; // solution scheme for CondFD - default
+    Real64 SpaceDescritConstant = 3.0;                                   // spatial descritization constant,
+    Real64 MinTempLimit = -100.0;                                        // lower limit check, degree C
```

Comment:
Are these not fully constant?  I would have expected that you could pull these out of state and make them constexpr, but maybe they can be overridden?  :shrug:

Reply:
These are not overridden. I'll pull `MinTempLimit` and `MaxTempLimit` out of state and make these `constexpr`. `SpaceDescritConstant` is overwritten by IDF input, so I'll not make it `constexpr`.